kano_settings/system/proxy.py

```diff
@@ -50,6 +54,63 @@ def set_chromium(enable, host=None, port=None):
     return
 
 
+def set_midori(enable, host=None, port=None, user_agent='Mozilla/5.0 (X11; Linux) AppleWebKit/535.22+ Midori/0.4'):
+
+    # Read Midori configuration file in
+    with open(midori_cfg, 'r') as f:
+        config_lines=f.readlines()
```

Comment:
Do we need to check that the file exists before just assuming that it will be there?